cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -140,12 +152,41 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
           statsOnlyResultTable()
         }
         p("For more details on primary and secondary server roles, see <<operations-manual#clustering-introduction-operational, Cluster overview>>.")
+        note {
+          p(
+            """The `TOPOLOGY` are only available to standard databases and not composite databases.
+              |Composite databases are always available on all servers.""".stripMargin)
+        }
+      }
+      section("Creating composite databases", "administration-databases-create-composite-database", "enterprise-edition") {
+        // TODO: add a link to the place explaining what composite databases are
+        p(
+          """Composite databases are a type of database that instead of containing data holds references to other databases that can be queried together through its constituent aliases.""".stripMargin)
+        p("Composite databases can be created using `CREATE COMPOSITE DATABASE`.")
+        query("CREATE COMPOSITE DATABASE inventory", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        note {
+          p("""Composite database names are subject to the same rules as standard databases.
+              |One difference is however that the deprecated syntax using dots without enclosing the name in backticks is not available.
+              |Both dots and dashes needs to be enclosed within backticks when using composite databases.""")
+        }
+        p("When a composite database has been created, it will show up in the listing provided by the command `SHOW DATABASES`.")
+        query("SHOW DATABASES YIELD name, type, access, role, writer, constituents", assertDatabasesShown) {
+          resultTable()
+        }
+        p(
+          """To create database aliases in the composite database the composite database is given as the namespace for the alias.
```

Comment:
A bit confusing this one too. Does it mean something like this?

```suggestion
          """To create database aliases in the composite database, you need to give the composite database the namespace for the alias.
```

Reply:
```suggestion
          """To create database aliases in the composite database, you need to give the composite database as namespace for the alias.
```

Reply:
Does that mean giving the same name of the composite database for the alias?

Reply:
the alias does not have the ame of the composite as its own name, it has it as the namespace for the name?

take procedures for example, the procedure is named `listConfig` and the namespace is `dbms` so the full/qualified name of the procedure is `dbms.listConfig`, it's the same principle for aliases in composite databases, the composite database name is the namespace for the alias

this means I can have 2 aliases with the same name in different namespaces:
CREATE ALIAS comp1.alias FOR ...
CREATE ALIAS comp2.alias FOR ...

Reply:
Right, so it's the same thing we're discussing in a previous comment. Let's go with your suggestion:

```suggestion
          """In order to create database aliases in the composite database, give the composite database as namespace for the alias.
```

Reply:
```suggestion
          """To create a database alias in a composite database the name of the composite database is given as namespace for the alias.
```

Reply:
the problem of not refreshing and therefore not seeing your reply XD